src/utils/localClient.ts

```diff
@@ -1,7 +1,4 @@
-// function wait(delay: number) {
-//   return new Promise(done => setTimeout(done, delay));
-// }
-
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Avoid using 'eslint-disable' for '@typescript-eslint/no-explicit-any'. Instead, try to refactor the code to avoid using 'any' type.